lib/CleantalkSP/SpbctWP/Scanner/ScannerQueue.php

```diff
@@ -1582,22 +1596,45 @@ public function send_results(){
                 $unknown[$path] = array_values($row);
             }
         }
-        
-        // Count files to scan
-        $scanned_total = $this->countFileSystem();
-        $scanned_total = $scanned_total['total'];
-        $error         = '';
-        
+
+        $error = '';
+
+        /**
+         * Getting parameters for security_mscan_logs()
+         */
+        $key = $spbc->settings['spbc_key'];
```

Comment:
May be to use Factory for creating object? For example securityMscanLogsFactory()

And use object in API::method__security_mscan_logs(factoryObj)

think about the names

Reply:
Yeah. It will be the great refactoring case.